.travis.yml

```diff
@@ -33,21 +33,21 @@ matrix:
     env: TOX_ENV="pep8,check_isort"
 
   - python: 2.7
-    env: TOX_ENV=py27
+    env: TOX_ENV=py27 TRIAL_FLAGS="-j 2"
 
   - python: 2.7
-    env: TOX_ENV=py27-old
+    env: TOX_ENV=py27-old TRIAL_FLAGS="-j 2"
 
   - python: 2.7
     env: TOX_ENV=py27-postgres TRIAL_FLAGS="-j 4"
     services:
       - postgresql
 
   - python: 3.5
-    env: TOX_ENV=py35
+    env: TOX_ENV=py35 TRIAL_FLAGS="-j 2"
 
   - python: 3.6
-    env: TOX_ENV=py36
+    env: TOX_ENV=py36 TRIAL_FLAGS="-j 2"
```

Comment:
Why do we use `-j 2` here rather than `-j 4` like we do for the postgres tests?

Reply:
I tested it, `-j 2` is faster for SQLite, which has less I/O blocking and therefore is approximate to the speed of your CPU. `-j 4` is better for Postgres which has a lot more I/O blocking (mostly creating the database).

Reply:
(Possibly a comment to that effect would be useful)

Reply:
done